src/features/authentication/login/login.scss

```diff
@@ -33,7 +26,7 @@
     border-radius: 14px;
   }
 
-  .input {
+  .login__input {
```

Comment:
Same suggestion as in the previous files, the syntax can be simplified to `&__input` and the same goes for all the classes here.

Reply:
Yep. After I saw the previous comment, I thought about changing the syntax everywhere.